data/hmm_params_real/IGHJ4_star_01.yaml

```diff
@@ -437,6 +437,7 @@ states:
     track: nukes
   extras: {ambiguous_char: N, ambiguous_emission_prob: 0.25, germline: N}
   name: insert_right_N
-  transitions: {end: 0.040000000000000036, insert_right_N: 0.96}
+  # Note that the "end" transition probability should be 0.040000000000000036.
```

Comment:
Duncan's J gene partis files have an `end` probability of 0.040000000000000036 in the `insert_right_N` state, BUT have an `end` probability of 0.04 in the last germline state before it.  Technically, the `end` probabilities should be identical and we have an assertion that checks this (https://github.com/matsengrp/linearham/blob/25-gname-pr/src/NPadding.cpp#L62). I modified our example J gene file just to get the code running, and commented that I made the modification.